components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -187,6 +187,7 @@ function matchFunctions.getExtraData(match)
 		team2icon = getIconName(opponent2.template or ''),
 		lastgame = Variables.varDefault('last_game'),
 		octane = match.octane,
+		ballchasing = match.ballchasing,
```

Comment:
Links should be stored in `match.links`, not `match.extradata` (octane is wrong, see #1141)

Reply:
They use slugs, not links

Reply:
https://liquipedia.net/commons/Help:LiquipediaDB/Guidelines#Social_media.2Fmisc_links

Reply:
They are both slugs that you can use with their APIs